### PR TITLE
Commenting post-install change implicatations for the sample inventories

### PR DESCRIPTION
**Pull request by brenton**

### CODE REVIEW

**Comment by brenton** [comment]
Once we know my comments are accurate I'll copy them to the other sample inventory files.

File: inventory/byo/hosts.ose.example [hunk 1]
```diff
@@ -225,10 +260,14 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 #
 # Signing secrets, used to authenticate sessions using
 # HMAC. Recommended to use secrets with 32 or 64 bytes.
+#
+# Changing this post deployment will invalidate all tokens.
```
Comment (brenton) [inline inventory/byo/hosts.ose.example]: @liggitt Are these two comments about session secrets accurate?

Comment (liggitt) [inline inventory/byo/hosts.ose.example]: it invalidates login sessions (in progress logins), not OAuth tokens

File: inventory/byo/hosts.ose.example [hunk 2]
```diff
@@ -201,6 +233,9 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # set RPM version for debugging purposes
 #openshift_pkg_version=-3.1.0.0
 
+# Explicitly set the tag that should be used for containerized installs
+#openshift_image_tag=v3.1.1.6
```
Comment (brenton) [inline inventory/byo/hosts.ose.example]: @dgoodwin, is this still the correct way to force a given image version?

**Comment by dgoodwin** [comment]
Still the correct way yes, just not needed anymore for most users.

**Comment by brenton** [comment]
I reviewed the current example hosts file and the current docs are even better than what this PR was originally trying to do.

**Pull request closed without merging**